test/boost/README.md

```diff
@@ -68,6 +68,42 @@ Boost tests can also be run using test.py - which is a script that provides
 a uniform way to run all tests in scylladb.git - C++ tests, Python tests,
 etc.
 
+## Execution with pytest
+
+To run test with pytest execute 
+```bash
+pytest test/boost
+```
+To execute only one file, provide the path filename
+```bash
+pytest test/boost/aggregate_fcts_test.cc
+```
+Since it's a normal path, autocompletion works in the terminal out of the box.
+
```

Comment:
I'm missing here an example of how to run a single test function. I know there's an example below, but it's in the middle of another discussion and I think here is the perfect spot for such an example - after explaining how to run all tests, and how to run all tests in a single file, the natural question is how to run a single test.

Reply:
done